sources/packages/forms/src/form-definitions/studentadditionaltransportationappeal.json

```diff
@@ -0,0 +1,1124 @@
+{
+  "title": "Appeals - Student Additional Transportation",
+  "name": "studentAdditionalTransportationAppeal",
+  "path": "studentadditionaltransportationappeal",
+  "type": "form",
+  "display": "form",
+  "tags": [
+    "common"
+  ],
+  "components": [
+    {
+      "key": "financialInformation",
+      "label": "Additional Transportation",
+      "input": false,
+      "tag": "p",
+      "attrs": [
+        {
+          "value": "",
+          "attr": ""
+        }
+      ],
+      "className": "category-header-medium primary-color",
+      "content": "Additional Transportation",
+      "type": "htmlelement",
+      "hideLabel": true,
+      "tags": [],
+      "conditional": {
+        "show": "",
+        "when": null,
+        "eq": ""
+      },
+      "properties": {}
+    },
+    {
+      "clearOnHide": false,
+      "key": "panel",
+      "input": false,
+      "title": "Additional Transportation Panel",
+      "theme": "default",
+      "tableView": false,
+      "components": [
+        {
+          "title": "Transportation Exception Content",
+          "labelWidth": "",
+          "labelMargin": "",
+          "theme": "default",
+          "tooltip": "",
+          "customClass": "formio-panel-unset",
+          "collapsible": false,
+          "hidden": false,
+          "hideLabel": true,
+          "disabled": false,
+          "modalEdit": false,
+          "key": "panel",
+          "tags": [],
+          "properties": {},
+          "customConditional": "",
+          "conditional": {
+            "json": "",
+            "show": "",
+            "when": null,
+            "eq": ""
+          },
+          "logic": [],
+          "attributes": {},
+          "overlay": {
+            "style": "",
+            "page": "",
+            "left": "",
+            "top": "",
+            "width": "",
+            "height": ""
+          },
+          "type": "panel",
+          "label": "Panel",
+          "breadcrumb": "default",
+          "tabindex": "",
+          "input": false,
+          "tableView": false,
+          "components": [
+            {
+              "label": "<strong>Which of the following best describes your situation?</strong>",
+              "optionsLabelPosition": "right",
+              "inline": false,
+              "tableView": false,
+              "values": [
+                {
+                  "label": "I don't require additional transportation allowance",
```

Comment:
This option was removed from the student application.

![image](https://github.com/bcgov/SIMS/assets/54600590/3d2960ac-4adf-4976-9a5e-1dc43c57a67d)